sms_sponsorship/tests/test_sms_compassion.py

```diff
@@ -19,12 +19,112 @@ class TestSmsCompassion(BaseSponsorshipTest):
 
     def setUp(self):
         super(TestSmsCompassion, self).setUp()
-        self.child = self.create_child('UG1119182')
-        self.child.hold_id.channel = 'sms'
+        self.child = self.env['compassion.child'].search([
+            ('id', '=', 5)
```

Comment:
Never browse by id in test, you never know what the id will be on another test database... Why did you change how it was done previously?

Reply:
It didn't work with create_child method 
(Error: IntegrityError: duplicate key value violates unique constraint "compassion_project_icp_id_uniq")